src/PowerShell/Microsoft.WinGet.Client.Cmdlets/Cmdlets/GetUserSettingCmdlet.cs

```diff
@@ -1,30 +1,31 @@
-﻿// -----------------------------------------------------------------------------
-// <copyright file="GetUserSettingsCmdlet.cs" company="Microsoft Corporation">
-//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
-// </copyright>
-// -----------------------------------------------------------------------------
-
-namespace Microsoft.WinGet.Client.Commands
-{
-    using System.Collections;
-    using System.Management.Automation;
-    using Microsoft.WinGet.Client.Common;
-    using Microsoft.WinGet.Client.Engine.Commands;
-
-    /// <summary>
-    /// Gets winget's user settings.
-    /// </summary>
-    [Cmdlet(VerbsCommon.Get, Constants.WinGetNouns.UserSettings)]
-    [OutputType(typeof(Hashtable))]
-    public sealed class GetUserSettingsCmdlet : PSCmdlet
-    {
-        /// <summary>
-        /// Gets the settings file contents.
-        /// </summary>
-        protected override void ProcessRecord()
-        {
-            var command = new UserSettingsCommand(this);
-            command.Get();
-        }
-    }
-}
+﻿// -----------------------------------------------------------------------------
+// <copyright file="GetUserSettingsCmdlet.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------------
+
+namespace Microsoft.WinGet.Client.Commands
+{
+    using System.Collections;
+    using System.Management.Automation;
+    using Microsoft.WinGet.Client.Common;
+    using Microsoft.WinGet.Client.Engine.Commands;
+
+    /// <summary>
+    /// Gets winget's user settings.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, Constants.WinGetNouns.UserSetting)]
```

Comment:
I don't see `Constants.cs` in the changes, which would be required to change the name and value here.

Reply:
Thanks, will have a look over this and have that updated too.